challenges/classes.js

```diff
@@ -1,7 +1,41 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
+class CuboidMaker {
 
+  constructor(attrs) {
+    this.length = attrs.length;
+    this.width = attrs.width;
+    this.height = attrs.height;
+  }
+
+  volume() {
+    return `The volume is ${this.length * this.width * this.height}`;
+  }
+
+  surfaceArea() {
+    return `The surface area ${2 * (this.length * this.width + this.length * this.height + this.width * this.height)}`;
+  }
+}
+
```

Comment:
Class component syntax with methods located inside the class are well implemented.